src/deb/control/postinst

```diff
@@ -81,5 +83,3 @@ case "$1" in
   ;;
 
 esac
-
-service [[artifactId]] stop || true
```

Comment:
yeesh. I wonder if this was an intentional workaround to accommodate some sort of deploy step that needed to happen manually. Like maybe we wanted salt to deploy the package and stop the service, but we didn't want it to start back up until after we'd done some sort of manual step. Not saying that's a practice we should continue, but there might be some hidden intent here that we need to uncover.

Reply:
Good point. @fcabrales-plos Do you know the rhino deploy process? Is there a reason installing the package stops the service?

Reply:
Just wanted to note that I feel unqualified to approve this.  I can only give a trust approval.